plugins/modules/packaging/os/pacman_key.py

```diff
@@ -0,0 +1,314 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2019, George Rawlinson <george@rawlinson.net.nz>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
+DOCUMENTATION = '''
+---
+module: pacman_key
+author:
+- George Rawlinson (@grawlinson)
+version_added: "3.2.0"
+short_description: Manage pacman's list of trusted keys
+description:
+- Add or remove gpg keys from the pacman keyring.
+notes:
+- Use full-length key ID (40 characters).
+- Keys will be verified when using I(data), I(file), or I(url) unless I(verify) is overridden.
+- Keys will be locally signed after being imported into the keyring.
+- If the key ID exists in the keyring, the key will not be added unless I(force_update) is specified.
+- I(data), I(file), I(url), and I(keyserver) are mutually exclusive.
+- Supports C(check_mode).
+requirements:
+- gpg
+- pacman-key
+options:
+    id:
+        description:
+            - The 40 character identifier of the key.
+            - Including this allows check mode to correctly report the changed state.
+            - Do not specify a subkey ID, instead specify the primary key ID.
+        required: yes
```

Comment:
YAML booleans should always be `true`/`false`.
```suggestion
        required: true
```